playout/libretime_liquidsoap/generate_liquidsoap_cfg.py

```diff
@@ -34,31 +36,31 @@ def generate_liquidsoap_config(ss):
         fh.write("ignore(%s)\n" % key)
 
     auth_path = os.path.dirname(os.path.realpath(__file__))
-    fh.write('log_file = "/var/log/airtime/pypo-liquidsoap/<script>.log"\n')
+    if log_filepath is not None:
+        fh.write(f'log_file = "{log_filepath.resolve()}"\n')
     fh.write('auth_path = "%s/liquidsoap_auth.py"\n' % auth_path)
     fh.close()
 
 
-def run():
-    logging.basicConfig(format="%(message)s")
+def run(log_filepath: Optional[Path]):
     attempts = 0
     max_attempts = 10
     successful = False
 
     while not successful:
         try:
-            ac = AirtimeApiClient(logging.getLogger())
+            ac = AirtimeApiClient(logger)
```

Comment:
I am doing this in some parts of the code, as I don't want to miss any logs, in the future, the api client should remove this attr and directly import a logger from a module.

Note that loguru.logger is compatible with the logging.Logger